src/ProxyManager/Generator/ParameterGenerator.php

```diff
@@ -88,8 +99,9 @@ public function generate()
     {
         return $this->getGeneratedType()
             . (true === $this->passedByReference ? '&' : '')
+            . ($this->variadic ? '...' : '')
```

Comment:
This stuff is about method definition:

```php
function foo (...$bar) {
```

What code is doing the call-time string?


```php
$this->foo(...$bar);
```